sdk/web-pubsub/web-pubsub/review/web-pubsub.api.md

```diff
@@ -156,10 +165,12 @@ export interface WebPubSubGroup {
 // @public
 export class WebPubSubServiceClient {
     constructor(connectionString: string, hubName: string, options?: HubAdminClientOptions);
-    constructor(endpoint: string, credential: AzureKeyCredential, hubName: string, options?: HubAdminClientOptions);
+    constructor(endpoint: string, credential: AzureKeyCredential | TokenCredential, hubName: string, options?: HubAdminClientOptions);
     readonly apiVersion: string;
     closeConnection(connectionId: string, options?: CloseConnectionOptions): Promise<void>;
     endpoint: string;
+    // (undocumented)
+    generateClientToken(options?: GenerateClientTokenOptions): Promise<import("./generated").ClientTokenResponse>;
```

Comment:
what the heck is this return type? I've never seen an inline import in api view...

Reply:
I don't know, but I'm sure it's because I didn't create a nice type in my convenience client. I'm fixing it.